Source/WebDriver/Session.cpp

```diff
@@ -3282,21 +3283,89 @@ void Session::emitEvent(const String& eventName, RefPtr<JSON::Object>&& body)
 bool Session::eventIsEnabled(const String& eventName, const Vector<String>&)
 {
     // https://w3c.github.io/webdriver-bidi/#event-is-enabled
-    HashSet<String> topLevelBrowsingContexts;
-    // FIXME Add support to subscribe to specific browsing contexts
-    // https://bugs.webkit.org/show_bug.cgi?id=282981
+    for (const auto& subscription : m_subscriptions) {
```

Comment:
Prefer to call this `m_eventSubscriptions`

Reply:
This seems super inefficient in the case where we have a lot of "cache misses". Can we keep a HashSet of String/AtomicString of event names for which there are *any* subscriptions and use that as an initial filter?

Reply:
I went with a `HashMap<eventname, count>`, as the same event name might be subscribed to multiple times (e.g. global + different contexts)